bundle/manifests/gpu-operator-certified.clusterserviceversion.yaml

```
@@ -84,8 +84,94 @@ metadata:
+                    "replacement": "accelerator_gpu_utilization",
+                    "sourceLabels": [
+                      {
+                        "__name__",
+                      }
+                    ],
+                    "targetLabel": "__name__"
+                  },
+                  {
+                    "action": "replace",
+                    "regex": "DCGM_FI_DEV_MEM_COPY_UTIL",
+                    "replacement": "accelerator_memory_used_bytes",
+                    "sourceLabels": [
+                      {
+                        "__name__",
+                      }
+                    ],
+                    "targetLabel": "__name__"
+                  },
+                  {
+                    "action": "replace",
+                    "regex": "DCGM_FI_DEV_FB_TOTAL",
+                    "replacement": "accelerator_memory_total_bytes",
+                    "sourceLabels": [
+                      {
+                        "__name__",
+                      }
+                    ],
+                    "targetLabel": "__name__"
+                  },
+                  {
+                    "action": "replace",
+                    "regex": "DCGM_FI_DEV_POWER_USAGE",
+                    "replacement": "accelerator_power_usage_watts",
+                    "sourceLabels": [
+                      {
+                        "__name__",
+                      }
+                    ],
+                    "targetLabel": "__name__"
+                  },
+                  {
+                    "action": "replace",
+                    "regex": "DCGM_FI_DEV_GPU_TEMP",
+                    "replacement": "accelerator_temperature_celcius",
+                    "sourceLabels": [
+                      {
+                        "__name__",
+                      }
+                    ],
+                    "targetLabel": "__name__"
+                  },
+                  {
+                    "action": "replace",
+                    "regex": "DCGM_FI_DEV_SM_CLOCK",
+                    "replacement": "accelerator_sm_clock_hertz",
+                    "sourceLabels": [
+                      {
+                        "__name__",
+                      }
+                    ],
+                    "targetLabel": "__name__"
+                  },
+                  {
+                    "action": "replace",
+                    "regex": "DCGM_FI_DEV_MEM_CLOCK",
+                    "replacement": "accelerator_memory_cloc_hertz",
+                    "sourceLabels": [
+                      {
+                        "__name__",
+                      }
+                    ],
+                    "targetLabel": "__name__"
+                  }
+                ],
+                "relabelings": [
+                  {
+                    "action": "replace",
+                    "replacement": "nvidia",
+                    "targetLabel": "vendor_id"
+                  }
+                ],
+              },
```

@yevgeny-shnaidman the `validate-csv` check is failing with below error message:
```
time="2025-07-16T16:54:29Z" level=error msg="failed to validate alm-example: invalid character ',' after object key"
```
The below change is needed.
```suggestion
                ]
              }
```

---

fixed